cwnTheHub/cwn-storybook-ui
# fix(button-component): update test coverage and add outline button

update the test coverage for the component

- test click action
- test style for the component
- test component displays without crashing
- check if component can be in disabled state

add regular, outline and empty variants of button

- add stories for the variants
